app/lib/simulator/Simulator.coffee

```diff
@@ -67,15 +67,27 @@ module.exports = class Simulator extends CocoClass
         @simulatingPlayerStrings = {}
         for team in ['humans', 'ogres']
           session = _.find(taskData.sessions, {team: team})
+          unless session
+            @trigger 'statusUpdate', "Error simulating game: find 'no team' sessions. Trying another game in #{@retryDelayInSeconds} seconds."
+            @simulateAnotherTaskAfterDelay()
+            return
           @simulatingPlayerStrings[team] = "#{session.creatorName or session.creator} #{session.team}"
         @trigger 'statusUpdate', "Setting up #{taskData.sessions[0].levelID} simulation between #{@simulatingPlayerStrings.humans} and #{@simulatingPlayerStrings.ogres}"
         #refactor this
         @task = new SimulationTask(taskData)
+        try
+          levelID = @task.getLevelName()
+        catch err
+          console.error err
+          alert 'no levelname'
```

Comment:
Let's take this `alert` out
```suggestion
```

Reply:
oh my god. how can i forget to take this out ?